amqp_influxdb/__init__.py

```diff
@@ -39,10 +42,31 @@ def __init__(self,
                  exchange,
                  routing_key,
                  message_processor,
-                 connection_parameters=None):
+                 connection_parameters={}):
+        """
+            AMQPTopicConsumer initialisation expects a connection_parameters
+            dict as provided by the __main__ of amqp_influx.
+        """
         self.message_processor = message_processor
 
-        connection_parameters = connection_parameters or {}
+        credentials = connection_parameters.get('credentials', {})
+        credentials_object = pika.credentials.PlainCredentials(
+            username=credentials.get('username', 'guest'),
```

Comment:
as i said, do `username=credentials.get('username') or 'guest'` 
instead of of `username=credentials.get('username', 'guest')`
and the same for password